api/envoy/config/api_listener/v2/api_listener.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package envoy.config.api_listener.v2;
+
+option java_outer_classname = "ApiListenerProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.envoy.config.api_listener.v2";
+
+option java_generic_services = true;
+
+import "envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto";
+
+import "gogoproto/gogo.proto";
+
+option (gogoproto.equal_all) = true;
```

Comment:
Please remove this. We don't need to import gogoproto anymore.

Reply:
Done.